### PR TITLE
Phrase suggestions

### LINKED ISSUE #28: Change timestamps back to snake case.

### LINKED ISSUE #48: Machine Learning to suggest newPhrases
Add :suggestions column to phrase table

Add base phrases that make sense to be the next suggested/predicted word in the sequence

Icebox: Potentially add mutation to insert suggestions to specific phrases by the user

### PR DESCRIPTION
**Pull request by cgaddis36**
### Description
Add in :suggestions column to Phrase table to house suggested phrases after they have been spoken

### Type of change
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Refactor (non-breaking change which maintains existing functionality)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 
#### Notes
Will need to migrate your db after merging

### CODE REVIEW

File: app/models/phrase.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 class Phrase < ApplicationRecord
   has_many :section_phrases
   has_many :sections, through: :section_phrases
+
+  validates_presence_of :expression
```
Comment (cgaddis36) [inline app/models/phrase.rb:5]: added in validation for expressions on Phrases

File: db/migrate/20200914155208_add_suggestions_to_phrases.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddSuggestionsToPhrases < ActiveRecord::Migration[5.2]
+  def change
+    add_column :phrases, :suggestions, :string, array: true, default: []
+  end
+end
```
Comment (cgaddis36) [inline db/migrate/20200914155208_add_suggestions_to_phrases.rb:5]: new migration adding suggestions to Phrase Table

File: app/models/phrase.rb (hunk 1 above)
Comment (sagemlee) [reply app/models/phrase.rb:5]: Nice this should beef up coverage even more.

**Review by sagemlee — APPROVED** [review]

File: app/models/phrase.rb (hunk 1 above)
Comment (EricLarson2020) [reply app/models/phrase.rb:5]: Nice Work

**Review by EricLarson2020 — APPROVED** [review]

**Pull request merged**